api/clients/config.go

```diff
@@ -8,22 +8,32 @@ import (
 )
 
 type EigenDAClientConfig struct {
-	// RPC is the HTTP provider URL for the Data Availability node.
+	// RPC is the HTTP provider URL for the EigenDA Disperser
 	RPC string
 
-	// The total amount of time that the client will spend waiting for EigenDA to confirm a blob
+	// Timeout used when making dispersals to the EigenDA Disperser
+	// TODO: we should change this param as its name is quite confusing
+	ResponseTimeout time.Duration
```

Comment:
agree, how about DispersalTimeout?

Reply:
I'm fine with doing that, but as I mentioned in the other comment https://github.com/Layr-Labs/eigenda/pull/780#discussion_r1806364950, do we have a policy for these breaking changes? Or are we fine with just doing it now?

Reply:
Note: we decided to do all the breaking changes in a future PR.